install/kubernetes/helm/istio/charts/sidecarInjectorWebhook/templates/clusterrole.yaml

```diff
@@ -14,4 +14,16 @@ rules:
   verbs: ["get", "list", "watch"]
 - apiGroups: ["admissionregistration.k8s.io"]
   resources: ["mutatingwebhookconfigurations"]
-  verbs: ["get", "list", "watch", "patch"]
+  verbs: ["*"]
+- apiGroups: ["extensions","apps"]
+  resources: ["deployments"]
+  resourceNames: ["istio-sidecar-injector"]
+  verbs: ["get"]
+- apiGroups: ["extensions"]
+  resources: ["deployments/finalizers"]
```

Comment:
Where is the finalizer set?  Is this something that was added upstream?

Reply:
I'm not sure what you mean by 'added upstream', this hasn't been pushed up there yet.

This is a kubernetes requirement for any account which sets blockOwnerDeletion.